corehq/apps/app_manager/templates/app_manager/app_view_release_manager.html

```diff
@@ -31,13 +31,16 @@
   {% initial_page_data 'build_profiles' app.build_profiles %}
   {% initial_page_data 'latest_version_for_build_profiles' latest_version_for_build_profiles %}
   {% initial_page_data 'latestReleasedVersion' latest_released_version %}
+  {% initial_page_data 'master_briefs' master_briefs %}{# linked apps only #}
+  {% initial_page_data 'multiple_masters' multiple_masters %}{# linked apps only #}
   {% initial_page_data 'practice_users' practice_users %}
   {% initial_page_data 'enable_practice_users' app.enable_practice_users %}
   {% initial_page_data 'intro_only' intro_only %}
   {% initial_page_data 'langs' app.langs %}
   {% initial_page_data 'latest_build_id' latest_build_id %}
   {% initial_page_data 'sms_contacts' sms_contacts %}
   {% initial_page_data 'confirm' confirm %}
+  {% initial_page_data 'upstream_url' upstream_url_template %}
```

Comment:
why not use `registerurl`?

Reply:
From later logic I think the answer is that this could be an absolute URL in the case of remote linked apps

Reply:
yep